demos/applications/tla2528_analog_input.cpp

```diff
@@ -0,0 +1,35 @@
+
+#include <array>
+#include <libhal-util/bit.hpp>
+#include <libhal-util/i2c.hpp>
+#include <libhal-util/serial.hpp>
+#include <libhal-util/steady_clock.hpp>
+#include <libhal/input_pin.hpp>
+#include <libhal/units.hpp>
+#include <libhal-expander/tla2528.hpp>
+
+#include <libhal-expander/tla2528_adapters.hpp>
+#include <resource_list.hpp>
```

Comment:
```suggestion
#include <array>

#include <libhal-util/bit.hpp>
#include <libhal-util/i2c.hpp>
#include <libhal-util/serial.hpp>
#include <libhal-util/steady_clock.hpp>
#include <libhal/input_pin.hpp>
#include <libhal/units.hpp>
#include <libhal-expander/tla2528.hpp>
#include <libhal-expander/tla2528_adapters.hpp>

#include <resource_list.hpp>
```

Suggestion made based on https://libhal.github.io/4.1/contributor_guide/style/#s143-include-order.